source/elements/oneMKL/source/domains/spblas/data_types/set_matrix_property.rst

```diff
@@ -28,23 +29,51 @@ Matrix properties
    holds all the given properties. A property can be set as a hint for backends
    to optimize some operations. Multiple properties can be set to the same handle.
 
+symmetric
+^^^^^^^^^
+
+Guarantees that the user-provided matrix data are symmetric, meaning the matrix
+is square, the user data contain both lower and upper triangular regions, and
+that its transpose is equal to itself.
+
+sorted
+^^^^^^
+
+Guarantees that the user-provided matrix data is fully sorted. The table below
+details the property for each matrix format:
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 20 80
+
+      * - Matrix format
+        - Description
+      * - CSR
+        - Guarantees that the column indices are sorted in ascending order for
+          a given row.
+      * - COO
+        - Guarantees that the indices are sorted by row then by column in
+          ascending order.
```

Comment:
```suggestion
        - Guarantees that the indices are sorted by rows then by columns
          within rows in ascending order.
```

Reply:
Done in https://github.com/uxlfoundation/oneAPI-spec/pull/583/commits/eef74a30ab15edbfed9e1d470913aec66ec23903